packages/engine/tests/units/data/csv/immutable/src/behaviors/test.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Ensures a dataset is immutable
+ */
+const behavior = (state, context) => {
+  context.data()["dataset.csv"][0][0];
```

Comment:
Are you supposed to be setting something here? I don't understand what this does

Reply:
Yes, while refactoring the test I forgot to assign a value.
```suggestion
  context.data()["dataset.csv"][0][0] = "4";
```